### PR TITLE
FIX linter issues AND add fr-on; fix rewrite configs; renamed files accordingly

### PR DESCRIPTION
**Pull request by jeroach**
So this PR is now doing more things:
- Update `consistent-eval` to the latest version of https://github.com/Brightspace/eslint-config-brightspace/releases
- Fix all our linter issues (mostly spacing, line endings, and method alpha ordering)

Original intent of PR follows:

[US125902](https://rally1.rallydev.com/#/42960320374d/dashboard?detail=%2Fuserstory%2F509629160936&fdp=true?fdp=true): [fr-ON] Quick Eval and Consistent Eval Web components

Quick Eval and the QE Widget are being handled here: https://github.com/BrightspaceHypermediaComponents/activities/pull/1612

Fixes done as per pattern of the above linked PR for `activities`:

- Added `fr-on`
- Fixed serge rewrite config
- Renamed `cy-gb` to `cy`
- Renamed `zh` to `zh-cn`

### CODE REVIEW

**Comment by bearfriend** [comment]
You might also consider swapping the localization mixin for the [new dynamic one](https://github.com/BrightspaceUI/core/blob/master/mixins/localize-mixin.md#example-2-dynamically-imported-resources).

**Comment by jeroach** [comment]
> You might also consider swapping the localization mixin for the [new dynamic one](https://github.com/BrightspaceUI/core/blob/master/mixins/localize-mixin.md#example-2-dynamically-imported-resources).

Good call, I'll add this

File: localize-consistent-evaluation.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { LocalizeDynamicMixin } from '@brightspace-ui/core/mixins/localize-dynamic-mixin.js';
+
+export const LocalizeConsistentEvaluation = superclass => class extends LocalizeDynamicMixin(superclass) {
+
+	static get localizeConfig() {
+		return {
+			importFunc: async lang => (await import(`./lang/${lang}.js`)).default
```
Comment (jeroach) [inline localize-consistent-evaluation.js:7]: This is the dynamic lang import change

The file needed to be moved outside of the `lang` folder as per the nature of dynamic importing.

File: package.json [hunk 2]
```diff
@@ -22,11 +22,11 @@
     "@webcomponents/webcomponentsjs": "^2",
     "@open-wc/testing": "^2",
     "@open-wc/testing-karma": "^3",
-    "babel-eslint": "^10",
+    "@babel/eslint-parser": "^7",
     "deepmerge": "^3",
     "es-dev-server": "^1.38.3",
     "eslint": "^6",
-    "eslint-config-brightspace": "^0.6.6",
+    "eslint-config-brightspace": "^0.13.1",
```
Comment (jeroach) [inline package.json:29]: Updated linter version

File: lang/fr-on.js [hunk 3]
```diff
@@ -0,0 +1,114 @@
+/* eslint quotes: 0 */
+
+export default {
```
Comment (jeroach) [inline lang/fr-on.js:3]: Add `fr-on`

**Review by jmurray-d2l — APPROVED** [review]
Makes sense, will need to test the entire thing :)

**Review by Scy-D2L — APPROVED** [review]
That's a lot to go through! So much reordering. 👀

**Pull request merged**